.github/workflows/on_pull_request.yml

```diff
@@ -22,6 +22,10 @@ jobs:
   onPullRequestJob:
     name: Verify code base when pull request is published/updated
     runs-on: macos-latest
+    permissions:
+      # Set permissions for ${{ secrets.GITHUB_TOKEN }}
+      # https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+      packages: read
```

Comment:
```suggestion
      contents: read
      packages: read
```

Reply:
This has now been done; set to `contents: read`